README.md

```diff
@@ -1,6 +1,6 @@
 # gRPC client for the Provenance Blockchain
 
-**Tip**: Refer to the [Cosmos Proto Docs](https://docs.cosmos.network/master/core/proto-docs.html) and
+**Tip**: Refer to the [Cosmos Proto Docs](https://docs.cosmos.network/v0.47/tooling/protobuf) and
```

Comment:
actually, i think we want to use `main` instead of `master`? else we'll have to correct docs every time we upgrade

Reply:
do you mean latest instead of versioned?